content/en/docs/tasks/traffic-management/locality-load-balancing/single-cluster-lb/index.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Single Multi-Zone Cluster Load Balancer Standalone Example
+description: This example demonstrates the traffic shifting between zones within a single cluster.
```

Comment:
What's the use case for this configuration? Why would anyone want to configure failover to another node in the same cluster?

Reply:
Some users only need a single multi-zone cluster in one region. They want to know how to achieve zone isolation that resources and instances of the app are isolated between zones while assure the availability and failure recovery. Within a cluster, if one zone is down, the users would want to know how to shift the workloads to another available zone and continue to operate.

Reply:
I see, makes sense. I think you need to say this in the doc. The way you've just stuck this doc next to the other ones seems very awkward. Maybe you should organize the _index.md page better, something like this:

Locality load balancing can be configured for either a multicluster or single multi-zone cluster environment. TODO: explain the different use cases at a high level ...

Follow the instructions corresponding to your use case:

- Multicluster
    - Before you begin
    - Locality failover
    - Locality weighted distribution
    - Cleanup
 - Single multi-zone cluster

Reply:
updated. Please review. Thank you.